src/chat/tools/getEnvironments.ts

```diff
@@ -25,6 +27,26 @@ export interface IGetEnvironmentsParameters {
 export class GetEnvironmentsTool extends BaseLanguageModelTool<IGetEnvironmentsParameters> {
   readonly name = "get_environments";
 
+  prepareInvocation(
+    options: LanguageModelToolInvocationPrepareOptions<IGetEnvironmentsParameters>,
+    token: CancellationToken,
```

Comment:
Why pass token if not referenced and this little bit isn't any async or anything actually cancelable?